src/makers/index.ts

```diff
@@ -0,0 +1,82 @@
+import bobcat from './bobcat'
+import customAntennas from './custom/antennas'
+import finestra from './finestra'
+import helium from './helium'
+import longAP from './longAP'
+import nebra from './nebra'
+import rak from './rak'
+import syncrobit from './syncrobit'
+import { LangType, supportedLangs } from '../utils/i18n/i18nTypes'
+import { HotspotMakerLangField } from './hotspotMakerTypes'
+
+export const Makers: Record<string, { id: number; supportEmail: string }> = {
+  bobcat,
+  finestra,
+  helium,
+  longAP,
+  nebra,
+  rak,
+  syncrobit,
+}
+
+export const HotspotMakerModels = {
```

Comment:
Couple things here,

1. We need to make sure we keep the original order of the list, which would be ordered by maker ID.
2. I don't see Helium in the hotspot selection list for some reason

Reply:
Maybe its because the syncrobit hotspots.ts file is named Helium?

Reply:
Ah yeah looks like thats why

Reply:
Never mind on the ordering, Coco confirmed alphabetical is ok. Could we make sure it stays sorted no matter the order these are exported in though?

Reply:
Great find on the syncrobit!

Reply:
The order is currently alphabetical.
```
export const HotspotModelKeys = Object.keys(
  HotspotMakerModels,
).sort() as HotspotType[]
```